src/database/db_build.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+const db_connection = require("./db_connection")
+const buildFile = fs.readFileSync(__dirname + "/db_build.sql", "utf8");
+
+  db_connection.query(buildFile, (error, result)=>{
```

Comment:
indentation.

are you using a a linter? you have eslint in your dependencies

Reply:
we will open another branch for that.
thank you.